src/components/motion/MotionController.h

```diff
@@ -66,9 +75,17 @@ namespace Pinetime {
       }
 
     private:
-      uint32_t nbSteps = 0;
+      Utility::CircularBuffer<uint32_t, stepHistorySize> nbSteps = {0};
       uint32_t currentTripSteps = 0;
 
+      void SetSteps(Days day, uint32_t steps) {
+        nbSteps[static_cast<std::underlying_type_t<Days>>(day)] = steps;
+      }
+
+      uint32_t GetSteps(Days day) const {
```

Comment:
Can this method be removed since `NbSteps` is the same?

Or maybe we should rename `NbSteps` to `GetSteps` instead?

Reply:
Yeah, those two really were the same, I removed `GetSteps`

Renaming `NbSteps` would have broken pretty much every single pull requests that involve watchfaces